### DISCUSSION TITLE
"System Name" or some kind of user-facing name

### DISCUSSION THREAD

**Discussion by yaleman** [kanidm/kanidm#857] [Ideas]
I know we have the domain name, ie `kanidm.example.com`, but I could see a need for a user-facing name like "Krabby's Super Secure IT Company" in things like the UI headers or (as was the impetus for this) the TOTP Service name.

Obviously happy to yeet!() this into the code and make it work, but if this isn't blocked for reasons:

- what should the config file identifier be? 
  - `ui_title`
  - `system_name`
  - `domain_name` (pretty close to `domain`, so ew)
- what would the default be? `Kanidm (<domain>)` maybe?

**Comment by Firstyear** [comment]
`domain_display_name` seems like a reasonable choice.

I think just duplicating "domain_name" is a reasonable default.

**Comment by Firstyear** [comment]
Isn't this implemented now @yaleman ?

**Reply by yaleman** [reply]
Yep
